front-end-web/src/Components/Layouts/NavBar.tsx

```diff
@@ -8,11 +8,13 @@ import AccountCircle from "@material-ui/icons/AccountCircle";
 import MenuIcon from "@material-ui/icons/Menu";
 import React from "react";
 import { User } from "firebase";
+import { statement } from "@babel/template";
```

Comment:
This isn't used. I would remove all of the unused imports. You can use Alt-Shift-O to do this automatically in VSCode.